### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by ghost**
add slack, email and php cs fixer with docker

### CODE REVIEW

**Review by dachic — COMMENTED** [review]
Can you please add a blank line at the end of your file ?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.php_cs.cache
```
Comment (vodkhard) [inline .gitignore]: This line should be in your gitignore global

**Review by vodkhard — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ghost) [reply .gitignore]: I have already put it there

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+language: php
+
+php:
+  - 7.2
```
Comment (vodkhard) [inline .travis.yml:4]: Add php 5.5

**Review by vodkhard — CHANGES_REQUESTED** [review]

**Review by vodkhard — DISMISSED** [review]

**Review by DrissBelaroussi — DISMISSED** [review]
Nice

**Review by vodkhard — APPROVED** [review]
Perfect

**Pull request merged**